ras_backstage/controllers/collection_instrument_controller.py

```diff
@@ -0,0 +1,34 @@
+import logging
+
+import requests
+from requests import RequestException
+from structlog import wrap_logger
+
+from ras_backstage import app
+from ras_backstage.exception.exceptions import ApiError
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+def upload_collection_instrument(survey_id, collection_exercise_id, file):
+    logger.debug('Uploading collection instrument', collection_exercise_id=collection_exercise_id)
+    url = f'{app.config["RAS_COLLECTION_INSTRUMENT_SERVICE"]}upload/{collection_exercise_id}'
+
+    classifiers = {}
+    if survey_id:
+        classifiers['SURVEY_ID'] = survey_id
+    if collection_exercise_id:
+        classifiers['COLLECTION_EXERCISE'] = collection_exercise_id
+    try:
+        response = requests.post(url, auth=app.config['BASIC_AUTH'], files={'file': file})
```

Comment:
I saw this is WIP but it's probably worth mentioning that we have a `request_handler` function in `ras_backstage/common/requests_handler` which can handle the `RequestException`